pkg/archive/changes_test.go

```diff
@@ -178,6 +178,7 @@ func TestChangesWithChanges(t *testing.T) {
 	sort.Sort(changesByPath(changes))
 
 	expectedChanges := []Change{
+		{"/dir1", ChangeModify},
```

Comment:
Since I'm not able to reproduce the bug, and the testcase works for me w/o this test change, I'd like @vbatts to verify that we should actually see `/dir1` in the list of changes.

Wish I would reproduce this - would love to debug some more to better understand it.  But if @vbatts is ok with the test change then I think the changes look good.